docs/storage-drivers/inmemory.md

```diff
@@ -7,11 +7,14 @@ keywords = ["registry, service, driver, images, storage,  in-memory"]
 <![end-metadata]-->
 
 
-# In-memory storage driver
+# In-memory storage driver (Testing Only)
 
-An implementation of the `storagedriver.StorageDriver` interface which uses local memory for object storage.
+The `inmemory` storage driver is useful for testing purposes. This driver is an
+implementation of the `storagedriver.StorageDriver` interface which uses local
+memory for object storage.  If you use this driver, you should use ramdisk on
+your filesystem over pure in-memory.
```

Comment:
I guess @stevvooe remark from above applies below as well.
Also, maybe this sentence can be enhanced?